app.js

```diff
@@ -0,0 +1,20 @@
+const { getAllTopics } = require("./controllers/topicController");
+const express = require("express");
+const app = express();
+
+const {
+  errorHandler404,
+  errorHandler500,
+} = require("./controllerErrorHandler");
+
+app.get("/api", (req, res) => {
+  res.status(200).send({ msg: "all ok" });
+});
+
+app.get("/api/topics", getAllTopics); //create router
+
+app.use("/api/topicz", errorHandler404);
```

Comment:
This will only account for if the path is specifically /api/topiz. I would like you to make it so it is more dynamic. Have you considered using app.all('/*) ?